example/src/main/java/com/stripe/example/activity/ConnectUSBankAccountActivity.kt

```diff
@@ -35,6 +71,14 @@ class ConnectUSBankAccountActivity : StripeIntentActivity() {
                 result.onSuccess {
                     viewModel.status
                         .postValue("Collecting bank account information for payment")
+                    launcher.presentWithPaymentIntent(
+                        publishableKey = settings.publishableKey,
+                        clientSecret = it.getString("client_secret"),
```

Comment:
nit

```suggestion
                        clientSecret = it.getString("secret"),
```

The example backend mobile app will return a json blob with `secret` instead of `client_secret`. I am assuming this is from your own server?

Reply:
Yes, this is for now interacting with a test server. Let's maybe keep the fields / endpoints from the test server (as it's stated on the top of the activity that this is not yet ready to be a reference) and update at the end of the implementation when the test server is ready?

Reply:
@jameswoo-stripe sample backend now supports the full collect bank account flow, so updated it to use `secret` as client secret field key.